cmd/server/cmd/crd_test.go

```diff
@@ -0,0 +1,174 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+
+	"istio.io/mixer/pkg/adapter"
+	pkgadapter "istio.io/mixer/pkg/adapter"
+	"istio.io/mixer/pkg/template"
+)
+
+var empty = ``
+
+var exampleAdapters = []pkgadapter.InfoFn{
+	func() adapter.BuilderInfo { return adapter.BuilderInfo{Name: "foo-bar"} },
+	func() adapter.BuilderInfo { return adapter.BuilderInfo{Name: "abcd"} },
+}
+var exampleAdaptersCrd = `
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    impl: foo-bar
```

Comment:
Please fix these templates per spec.
After you generate one,
please run `kubectl create -f   mixer-crd.yaml`